app/views/appointment_summaries/new.html.erb

```diff
@@ -1,3 +1,4 @@
+<% content_for(:page_title, 'How to view your appointment summary - Pension Wise') %>
```

Comment:
Can we move these into the locale file as per the other pages?

Also just noticed that the F2F booking pages are all missing page titles too!